CMakeLists.txt

```diff
@@ -173,7 +174,7 @@ if (NGRAPH_CPU_ENABLE)
     set(NGRAPH_INTEL_CPU_ONLY_ENABLE ON)
 endif()
 
-if (NGRAPH_MLIR_ENABLE OR NGRAPH_CODEGEN_ENABLE)
+if (NGRAPH_MLIR_ENABLE OR NGRAPH_CODEGEN_ENABLE OR NGRAPH_CPU_MLIR_ENABLE)
```

Comment:
This may be a latent bug, but should this say "NGRAPH_CPU_CODEGEN_ENABLE" where it now says "NGRAPH_CODEGEN_ENABLE"?